Modules/Core/Common/include/itkMapContainer.h

```diff
@@ -275,6 +275,9 @@ class ITK_TEMPLATE_EXPORT MapContainer
     ConstIterator(const Iterator & r)
       : m_Iter(r.m_Iter)
     {}
+    ConstIterator(const ConstIterator & r)
```

Comment:
Thank you Davis, But could this copy-constructor simply be defaulted? Could both copy-constructor and assignment operator be defaulted _implicitly_, according to the Rule Of Zero? If so, that would be my preference.

Reply:
@N-Dekker Thank you for the review!  You're absolutely right--please see new commits.